crawler/WikiSpider.py

```diff
@@ -31,6 +30,16 @@ def arg_str2dict(arg):
     return arg_dict
 
 
+def arg_contains_output_and_silent(args):
+    try:
+        args['output']
+        args['silent']
+    except Exception:
+        return False
+    else:
+        return True
+
+
```

Comment:
I'm sorry, but I cannot agree with this variant.

As far as I understood, below you can find better replacement for `#if output=stdout` condition:
```
if self.args and 'output' in self.args:
    if self.args['output'] == 'stdout' and 'silent' in self.args:
        pass
```